src/spclient.rs

```diff
@@ -101,20 +291,17 @@ impl SpClient {
         scan_sk: SecretKey,
         spend_key: SpendKey,
         mnemonic: Option<String>,
-        birthday: u32,
         is_testnet: bool,
```

Comment:
Not necessarily related to this PR, but I would like to remove all `is_testnet` with a `Bitcoin::Network` type. Then we can also use SpClient for regtest.

Reply:
Agree, I was just thinking about that too, I'll do it asap and let's ship it in this PR